src/cli.ts

```diff
@@ -133,10 +138,57 @@ async function run() {
       },
       exportWmafResultToCsv
     )
+    .command(
+      'vmaf',
+      'Use pipeline to run VMAF for the given distorted and reference files',
+      (yargs) => {
+        return yargs.options({
+          pipeline: {
+            type: 'string',
+            description: 'Path to pipeline file',
+            required: true
+          },
+          reference: {
+            type: 'string',
+            description: 'Uri or path to reference file',
+            required: true
+          },
+          distorted: {
+            type: 'string',
+            description: 'Uri or path to distorted file',
+            required: true
+          },
+          model: {
+            type: 'string',
+            description: 'VMAF model to use',
+            default: 'HD'
+          }
+        });
+      },
+      runVmaf
+    )
     .help()
     .parse();
 }
 
+async function runVmaf(argv) {
+  const reference = argv.reference;
+  const distorted = argv.distorted;
+  const pipeline = await loadPipeline(argv.pipeline);
+  const qualityFile = argv.distorted.replace('.mp4', '_vmaf.json');
```

Comment:
Will we only allow running on mp4 files?

Reply:
Not really this was copy pasted from other part of the code. Will fix.

Reply:
Fixed.